xla/service/gpu/gpu_latency_hiding_scheduler.cc

```diff
@@ -139,18 +139,18 @@ bool IsAsyncPair(const HloInstruction& from, const HloInstruction& target) {
 }
 
 // Count the maximum overlapping count in subgroups of group and other
-size_t CountOverlappingRanks(const std::vector<ReplicaGroup>& group,
-                             const std::vector<ReplicaGroup>& other) {
+size_t CountOverlappingRanks(const std::vector<std::vector<int64_t>>& group,
+                             const std::vector<std::vector<int64_t>>& other) {
   size_t overlapping_count = 0;
   for (const auto& curr_replica_group : group) {
     absl::flat_hash_set<int> curr_replica_ids;
-    for (const auto curr_replica_id : curr_replica_group.replica_ids()) {
+    for (const auto curr_replica_id : curr_replica_group) {
```

Comment:
Since you're touching this line can you specify type explicitly for readability?